src/app/business/home/home.component.ts

```diff
@@ -280,7 +291,8 @@ export class HomeComponent implements OnInit {
             this.selectedType = null;
             let fs_arr = this.Allbackends['fusionstorage-object'] ? this.Allbackends['fusionstorage-object'] : [];
             let ceph_arr = this.Allbackends['ceph-s3'] ? this.Allbackends['ceph-s3'] : [];
-            this.typeDetail = fs_arr.concat(ceph_arr);
+            let yig_arr = this.Allbackends['yig'] ? this.Allbackends['yig'] : [];
+            this.typeDetail = fs_arr.concat(ceph_arr,yig_arr);
```

Comment:
Why yig and ceph backends are concat together?

Reply:
Since both ceph_arr and yig_arr need to concat into fs_arr, we can concat at the same time.

Reply:
Ok. I see it now.